xxhsum.c

```diff
@@ -658,7 +658,7 @@ static int XSUM_hashFile(const char* fileName,
         XSUM_setBinaryMode(stdin);
     } else {
         if (XSUM_isDirectory(fileName)) {
-            XSUM_log("xxhsum: %s: Is a directory \n", fileName);
+            XSUM_log("xxhsum: %s: Is a directory\n", fileName);
```

Comment:
These space characters are intentional.

They make the printed line more readable
when the printing overwrites a screen's section which is already filled with garbage characters.

I recommend removing this file's modifications from your PR.